Corona-Warn-App/src/test/java/de/rki/coronawarnapp/appconfig/mapping/CWAConfigMapperTest.kt

```diff
@@ -237,34 +264,7 @@ class CWAConfigMapperTest : BaseTest() {
     }
 
     @Test
-    fun `feature, unencrypted checkins disabled`() {
```

Comment:
I removed some test cases like this because they were covered by other tests above already.